### DISCUSSION TITLE
Questions about notification properties

### DISCUSSION THREAD

**Discussion by Mikenux** [flatpak/xdg-desktop-portal#1511] [Questions]
- Why is `system.custom-alert` a `purpose` and not an independent property or a `display-hint`?
- Regarding categories, what does "to display the notification specially" mean?

**Comment by jsparber** [comment]
> Why is system.custom-alert a purpose and not an independent property or a display-hint?

From an app perspective, this is an action similar to a button. The only difference to a normal button is that the action isn't preformed by the user, but by the system automatically based on the platforms' policy.

 > Regarding categories, what does "to display the notification specially" mean?

 It means that a notification server can style a notification differently, and/or play a different sound, based on the category.  For example, an extreme weather notification may be colored red, displayed full screen and play an alert sound to ensure that the user sees the warning.

**Reply by Mikenux** [reply]
> From an app perspective, this is an action similar to a button. The only difference to a normal button is that the action isn't preformed by the user, but by the system automatically based on the platforms' policy.

Ok.

> It means that a notification server can style a notification differently, and/or play a different sound, based on the category. For example, an extreme weather notification may be colored red, displayed full screen and play an alert sound to ensure that the user sees the warning.

I still don't agree naming things that could not be what they are if they are present in system settings. However, I think that you could be a bit more precise instead of using "specially": "in a custom fashion" or "with a custom appearance", something like that.

Also, because you mentioned a cell broadcast notification, is there anything planned since apps would need to have access to the cellular network to use them?

And for "os.battery.low" which is logically system-reserved?
